common/hyperscaler/rules/definitions.go

```diff
@@ -0,0 +1,162 @@
+package rules
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/kyma-project/kyma-environment-broker/internal/broker"
+)
+
+const (
+	PR_ATTR_NAME = "PR"
+	HR_ATTR_NAME = "HR"
+	EU_ATTR_NAME = "EU"
+	S_ATTR_NAME  = "S"
+
+	HYPERSCALER_LABEL = "hyperscalerType"
+	EUACCESS_LABEL    = "euAccess"
+	SHARED_LABEL      = "shared"
+
+	ASTERISK                 = "*"
+	ATTRIBUTE_WITH_VALUE     = "attr"
+	SIGNATURE_ATTR_SEPARATOR = ":"
+)
+
+var InputAttributes = []Attribute{
+	{
+		Name:            PR_ATTR_NAME,
+		Description:     "Platform Region",
+		Setter:          setPlatformRegion,
+		Getter:          func(r *Rule) string { return r.PlatformRegion },
+		MatchableGetter: func(r *MatchableAttributes) string { return r.PlatformRegion },
+		input:           true,
+		output:          false,
+		HasValue:        true,
+		ApplyLabel: func(r *Rule, labels map[string]string) map[string]string {
+			if labels[HYPERSCALER_LABEL] == "" {
+				labels[HYPERSCALER_LABEL] = r.Plan
+			} else {
+				labels[HYPERSCALER_LABEL] += "_" + r.PlatformRegion
+			}
+			return labels
+		},
+	},
+	{
+		Name:        HR_ATTR_NAME,
+		Description: "Hyperscaler Region",
+		Setter:      setHyperscalerRegion,
+		Getter:      func(r *Rule) string { return r.HyperscalerRegion },
+		input:       true,
+		output:      false,
+		HasValue:    true,
+
+		ApplyLabel: func(r *Rule, labels map[string]string) map[string]string {
+			if labels[HYPERSCALER_LABEL] == "" {
+				labels[HYPERSCALER_LABEL] = r.Plan
+			} else {
+				labels[HYPERSCALER_LABEL] += "_" + r.HyperscalerRegion
+			}
+			return labels
+		},
+	},
+}
+
+var OutputAttributes = []Attribute{
+	{
+		Name:        EU_ATTR_NAME,
+		Description: "EU Access",
+		Setter:      setEuAccess,
+		Getter:      func(r *Rule) string { return strconv.FormatBool(r.EuAccess) },
```

Comment:
Why don't we just return boolean?
Instead of string representation of boolean.

Reply:
Because platform and cluster region use strings as values. It is done so to keep it the same.

Reply:
So if we have some attributes with string value, all attributes must be string?

Reply:
It all done to keep definitions extensible for the future as requested by PK in his implementation.

Reply:
We make this particular part extensible (but in a wrong way) keeping other parts (mirroring) closed for extension.